src/constants.js

```diff
@@ -0,0 +1,14 @@
+var exports = module.exports = {};
+
+/**
+ * Object containing the Auth0 domain for each region server.
+ */
+exports.DOMAINS_BY_REGION = {
+  us: 'login.auth0.com',
+  eu: 'login.eu.auth0.com'
+};
```

Comment:
That's what I thought. Left it there because the already existing SDK v2 code used it. The API should be tenant.auth0.com where "tenant" is the user account, right?